tests/orm/utils/test_node.py

```diff
@@ -23,3 +23,48 @@ def test_load_node_class_fallback():
     with pytest.warns(UserWarning):
         loaded_class = load_node_class('__main__.SubData.')
     assert loaded_class == Data
+
+
+# Test for Remove_prefix
+
+
+def test_load_node_class_with_node_prefix():
+    """Test the behavior of load_node_class with node prefix."""
+    # Test node prefix removal
+    with pytest.warns(UserWarning):
+        loaded_class = load_node_class('node.data.dict.')
+        assert loaded_class == Data
```

Comment:
Let's put the asserts outside of the context manager (same below)
```suggestion
    assert loaded_class == Data
```

Reply:
I think this would cause a syntax error

Reply:
Why would it cause syntax error?

Reply:
Apologies ! I didn't see that line of the code I thought the indentation problem but it isn't there